watson-developer-cloud/discovery-components
# fix: footer title field length expands parent component too much

#### What do these changes do/fix?

This fix prevents the footer title length from expanding its parent so much where it push the Improvement and Customize Panel component off the screen in tooling.

#### How do you test/verify these changes?

This is a companion PR with (https://github.ibm.com/Watson-Discovery/discovery-tooling/pull/7397).  To test this code, checkout this PR in components and the other PR in tooling, and use `yarn link-components`.

To verify the change works, load the sample project and search `IBM`.  See the search results.  Inspect the footer title of the first result.  Manually edit the text information with the footer tag to something very long.  See that the footer title will have an `...` at the end and does not push the Improvement and Customize Panel off the screen.

Before:

![8cd6a500-5cb3-11ea-96ca-30f68119ca62](https://user-images.githubusercontent.com/32341300/79011395-8bf22800-7b29-11ea-9670-a86afbb1815d.png)

After:

![Screen Shot 2020-04-10 at 10 12 49 AM](https://user-images.githubusercontent.com/32341300/79011403-957b9000-7b29-11ea-9f7a-f81934f6c81c.png)
